src/main.js

```diff
@@ -1,17 +1,20 @@
 import { ValidatorPollingService } from './validator-polling-service.js';
-import { SMSNotifier } from './notifiers/index.js';
+import { SMSNotifier, TelegramNotifier } from './notifiers/index.js';
 import { config } from './config.js';
 import { BeaconAPIClient } from './beacon-api-client.js';
 import { validatorBalanceReducedAlert, validatorStatusChangedAlert } from './alerts/index.js';
 import { logger } from './logger.js';
 
-const smsNotifer = new SMSNotifier(config.sms)
+const smsNotifier = new SMSNotifier(config.sms);
+const telegramNotifier = new TelegramNotifier(config.telegram);
```

Comment:
I think it would be useful to only create the notifier if a configuration is provided, e.g. if I only want telegram but not SMS, otherwise the app may end up crashing or creating a lot of error if any of the config is not provided:

```suggestion
const notifiers = [];
if (config.sms) notifiers.push(new SMSNotifier(config.sms));
if (config.telegram) notifiers.push(new TelegramNotifier(config.telegram));

if (notifiers.length === 0) {
  throw new Error('Missing notifier config, check your .env file!');
}
```

Reply:
^^ this way it should support having just one or both types of notifiers

Reply:
fully agree with you, I was considering having it done in a similar way. As I want to add quick MS Teams integration for personal reasons :) and it would be nice to have an elegant way to manage what notifications will be applied.